### DISCUSSION TITLE
KHR_materials_variants producing wrong JSON tag

### DISCUSSION THREAD

**Discussion by teammixtive** [google/model-viewer#2478] [Q&A]
When using the KHR_materials_variants extension when combining several GLB-files, I've found that the output GLB contains the tag "mapping", but the required tag by model-viewer is "mappings". The issue has been raised in facebookincubator/glTFVariantMeld (that Khronos recommends), but has not yet been addressed. 

Any thoughts on how to handle this? What I do now is simply editing the GLB file by hand in order for it to work... 

Thanks!
![image](https://user-images.githubusercontent.com/72607050/121928528-881af080-cd40-11eb-882c-25f53f1dc593.png)

**Comment by elalish** [comment]
Can you link to the issue? I can ping the formats group about it.

**Reply by teammixtive** [reply]
Of course! 
https://github.com/facebookincubator/glTFVariantMeld/issues/32
